composer.json

```diff
@@ -11,7 +11,11 @@
         "symfony/process": "^4.0",
         "symfony/options-resolver": "^4.0",
         "guzzlehttp/guzzle": "^6.0",
-        "webmozart/assert": "^1.4"
+        "webmozart/assert": "^1.4",
```

Comment:
please sort them alphabetically (there is a composer setting for it which I set globally)

Reply:
or we use ergebnis/composer-normalize to normalize the composer.json file

Reply:
if not done globally, this is the local setting for it but I don't want to have another dependency added:
```
    "config": {
        "optimize-autoloader": true,
        "sort-packages": true
    },
```